tests/functional/diff/__init__.py

```diff
@@ -16,6 +16,7 @@
 import sys
 import time
 import uuid
+from distutils import dir_util
```

Comment:
Import order is not proper here. Not in this line, and not the line directly below.